app/views/admin/responses/_attachments.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if attachable.can_order_attachments? && attachable.attachments.many? %>
```

Comment:
Now we've cleaned up the partial so it works with both we can render it and pull the dupe code out

```
    <%= render('/attachments/attachments', attachable: attachable) if attachable.attachments.any? %>

```